systems_cross_building/cross_building.rst

```diff
@@ -157,22 +157,41 @@ Windows to Raspberry PI
 
 .. code-block:: text
 
+    target_host=arm-linux-gnueabihf
+    standalone_toolchain=C:/sysgcc/raspberry
+    cc_compiler=gcc
+    cxx_compiler=g++
+
     [settings]
     os_build=Windows
     arch_build=x86_64
     os=Linux
-    arch=armv7hf
+    arch=armv7
     compiler=gcc
-    compiler.version=6.3  # We have chosen gcc 6.3 installer
+    compiler.version=6
     compiler.libcxx=libstdc++11
     build_type=Release
 
     [env]
-    CC=arm-linux-gnueabihf-gcc
-    CXX=arm-linux-gnueabihf-g++
+    CONAN_CMAKE_FIND_ROOT_PATH=$standalone_toolchain/$target_host/sysroot
+    PATH=[$standalone_toolchain/bin]
```

Comment:
If PATH is defined and most tools (AR, LD, STRIP) are in that path, is it always necessary to define the below env-vars?

Reply:
No, but it is harmless and covers all cases.